examples/jsm/renderers/common/Pipelines.js

```diff
@@ -67,7 +67,7 @@ class Pipelines extends DataMap {
 
 			if ( pipeline === undefined ) {
 
-				if ( previousPipeline && previousPipeline.usedTimes === 0 ) this._releasePipeline( computeNode );
+				if ( previousPipeline && previousPipeline.usedTimes === 0 ) this._releasePipeline( previousPipeline );
```

Comment:
I think this is just a typo? `this._releasePipeline` takes pipelines and uses their `cacheKey` property to free them from `this.caches`.